pytket/tests/placement_test.py

```diff
@@ -100,31 +109,10 @@ def test_placements_serialization() -> None:
 
 
 def test_placement_config() -> None:
-    test_coupling = [(0, 1), (1, 2), (1, 3), (4, 1), (4, 5)]
+    test_coupling = [(0, 1)]
     test_architecture = Architecture(test_coupling)
     test_pl = GraphPlacement(test_architecture)
-    test_circuit = Circuit(6)
-    test_circuit.CX(0, 1)
-    test_circuit.CX(2, 3)
-    test_circuit.CX(4, 3)
-    test_circuit.CX(2, 4)
-    test_circuit.CX(3, 5)
-    test_circuit.CX(0, 5)
-    circ1 = test_circuit.copy()
-    circ2 = test_circuit.copy()
-    map1 = test_pl.get_placement_map(test_circuit)
-    test_pl.place(circ1)
-    test_pl.modify_config(
-        max_matches=1, depth_limit=0, max_interaction_edges=2, timeout=100
-    )
-    map2 = test_pl.get_placement_map(test_circuit)
-    test_pl.place(circ2)
-    assert map1 != map2
-
-    mm = MappingManager(test_architecture)
-    mm.route_circuit(circ1, [LexiLabellingMethod(), LexiRouteRoutingMethod()])
-    mm.route_circuit(circ2, [LexiLabellingMethod(), LexiRouteRoutingMethod()])
-    assert circ1.n_gates < circ2.n_gates
+    assert test_pl.modify_config() == None
```

Comment:
```suggestion
    assert test_pl.modify_config() is None
```
(This is a better test since `==` can always be overridden.)
But I think we should include a bit more in this test -- just to make sure `test_pl.place()` still works after this. There may be user (or extensions) code that is doing this and we don't want to break it.

Reply:
updated test!